doc/parameters/param_list.md

```diff
@@ -479,6 +479,16 @@ When `planner_plugins` parameter is not overridden, the following default plugin
 | ----------| --------| ------------|
 | stop_on_failure | true | Whether to fail action task if a single waypoint fails. If false, will continue to next waypoint. |
 | loop_rate | 20 | Rate to check for results from current navigation task |
+| waypoint_task_executor_plugin | `waypoint_task_executor` | Name of plugin to be loaded for executing waypoint tasks.|
+
+## waypoint_task_executor plugin
```

Comment:
I think this is `WaitAtWaypoint` not `waypoint_task_executor`, since this is for a specific implementation of a waypoint task executor.

Reply:
Please refer here; https://github.com/ros-planning/navigation2/pull/1993/commits/4ad6f4dc2887dbdaf3d5aac557f351b74539af59#r499310248
I think it should be ok there ?

Reply:
No. `waypoint_task_executor` was correct and you didn't change the header name.